### PR TITLE
fix(vm): Fix IPv6 handling

### PR DESCRIPTION
**Pull request by RemkoMolier**
Attached a small bugfix for the bug in the handling of IPv6 configuration during vm creation I stumbled upon.

### Contributor's Note
Please mark the following items with an [x] if they apply to your PR.
Leave the [ ] if they are not applicable, or if you have not completed the item.
- [ ] I have added / updated documentation in `/docs` for any user-facing features or additions.
- [ ] I have added / updated templates in `/examples` for any new or updated resources / data sources.
- [x] I have ran `make examples` to verify that the change works as expected. 

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

### CODE REVIEW

**Comment by RemkoMolier** [comment]
Attached a small bugfix for the bug in the handling of IPv6 configuration during vm creation I stumbled upon.

File: example/resource_virtual_environment_vm.tf [hunk 1]
```diff
@@ -42,6 +42,9 @@ resource "proxmox_virtual_environment_vm" "example_template" {
       ipv4 {
         address = "dhcp"
       }
+      # ipv6 {
```
Comment (bpg-dev) [inline example/resource_virtual_environment_vm.tf:45]: Is there any particular reason for commenting `ipv6` here?

**Review by bpg — COMMENTED** [review]
Hi @RemkoMolier 👋🏼 
Thanks, good catch!

File: example/resource_virtual_environment_vm.tf (hunk 1 above)
Comment (bpg-dev) [reply example/resource_virtual_environment_vm.tf:45]: I mean, it should work even if uncommented.

Comment (bpg) [reply example/resource_virtual_environment_vm.tf:45]: nvm, I'll just merge it as-is

**Review by bpg — APPROVED** [review]
LGTM! 🚀

**Pull request merged**